config.json

```
@@ -15,6 +15,20 @@
     "indent_size": 4
   },
   "test_pattern": ".*[.]test[.]ts$",
+  "files": {
+    "solution": [
+      "%{kebab_slug}.ts"
+    ],
+    "test": [
+      "%{kebab_slug}.test.ts"
+    ],
+    "example": [
+      "%{kebab_slug}.example.ts"
```

Since the CI is broken anyway because of configlet lint, I rather fix this now so I don't have to manually update all the paths later. 

```suggestion
      ".meta/proof.ci.ts"
```